Source/wayland/GNUmakefile

```diff
@@ -0,0 +1,50 @@
+#
+#  Main makefile for GNUstep Backend win32
+#
+#  Copyright (C) 2002 Free Software Foundation, Inc.
```

Comment:
We probably should change the year and the name.

Reply:
Also did `s/win32/wayland`, and added Sergio and Ladislav.